addons/website_sale/static/src/js/website_sale.js

```diff
@@ -414,56 +414,21 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
      * @override
      * @private
      */
-    _updateProductImage: function ($productContainer, displayImage, productId, productTemplateId, new_carousel, isCombinationPossible) {
-        var $img;
+    _updateProductImage: function ($productContainer, displayImage, productId, productTemplateId, newCarousel, isCombinationPossible) {
```

Comment:
What makes `newCarousel` be always truthy now ?

Reply:
`get_combination_info_website` always returns it, but it was introduced mid v12 so the JS fix had to take into account the cases where the server would not be restarted and the RPC didn't return it yet.